tests/e2e/specs/customizer/header-builder/Site-identity/site-tagline.test.js

```diff
@@ -0,0 +1,59 @@
+import { createURL } from '@wordpress/e2e-test-utils';
+import { setCustomize } from '../../../../utils/set-customize';
+
+describe( 'Site Tagline Typography settings and color settings in the customizer', () => {
+	it( 'site tagline typography and color should apply corectly', async () => {
+		const siteTagline = {
+			'display-site-tagline-responsive': {
+				// eslint-disable-next-line quote-props
+				'desktop': true,
+				// eslint-disable-next-line quote-props
+				'tablet': true,
+				// eslint-disable-next-line quote-props
+				'mobile': true,
```

Comment:
```suggestion
			'display-site-tagline-responsive': {
				'desktop': true,
				'tablet': true,
				'mobile': true,
```
Let's not disable the rules, I think this is not required anyway and this is already fixed. Can you try and remove this?